testing/mocks/generic.go

```diff
@@ -435,6 +435,29 @@ func GenericAmount(delta int) cadence.Value {
 	return cadence.NewUInt64(random.Uint64())
 }
 
+func GenericSeals(number int) []*flow.Seal {
+	var seals []*flow.Seal
+	for i := 0; i < number; i++ {
+
+		seal := flow.Seal{
+			BlockID:    GenericIdentifier(2 * i),
+			ResultID:   GenericIdentifier(2*i + 1),
+			FinalState: GenericCommit(i),
+
+			AggregatedApprovalSigs: nil,
+			ServiceEvents:          nil,
```

Comment:
Should I mock `AggregatedApprovalSigs` and `ServiceEvents`?

Reply:
I would say it's not needed, since with the finalState/resultID/BlockID values we have more than enough data to be able to make assertions in the tests.